static/styles/subscriptions.css

```diff
@@ -620,6 +626,10 @@ form#add_new_subscription {
             margin-right: 8px;
         }
     }
+
+    .subscriber_list_add {
+        margin-bottom: 100px;
```

Comment:
Is this required ?
I couldn't see any difference on removing this.

Reply:
Ah! I thought it was working when I added it.